templates/widget/template/model.xml

```diff
@@ -19,7 +19,9 @@
             </property> -->
         </properties>
         <tags>
-
+            <% _.each(tags, function(tag){ %>
+            <tag type="<%= tag%>"></tag>
```

Comment:
Move the <%= tag %> as the value of the tag and have the type as 'regular' by default.